compile.js

```diff
@@ -206,6 +206,9 @@ qx.Class.define("qxl.apiviewer.compile.LibraryApi", {
           if (!res.apiindex.index[name]) {
             res.apiindex.index[name] = [];
           }
+          if (typeIdx === 2 && res.apiindex.index[name].length){
+            return;
+          }
           res.apiindex.index[name].push([typeIdx, nameIdx]);
         };
 
```

Comment:
Disregard. It's an in-place sort.

Reply:
@derrell there is a comment above the lines. I think it is not my concern.

Reply:
```
// We sort the result so that we can get a consistent ordering for loading classes, otherwise the order in
//  which the filing system returns the files can cause classes to be loaded in a slightly different sequence;
//  that would not cause a problem, except that the build is not 100% repeatable.
```